pkg/controller/ingress-shim/sync.go

```diff
@@ -109,7 +109,9 @@ func (c *controller) Sync(ctx context.Context, ing *extv1beta1.Ingress) error {
 
 func (c *controller) validateIngress(ing *extv1beta1.Ingress) []error {
 	var errs []error
+	namedSecrets := make(map[string]int)
 	for i, tls := range ing.Spec.TLS {
+		namedSecrets[tls.SecretName] = namedSecrets[tls.SecretName] + 1
```

Comment:
```suggestion
		namedSecrets[tls.SecretName] += 1
```
nit: might this notation be cleaner?